libs/feature/search/src/lib/fuzzy-search/fuzzy-search.component.ts

```diff
@@ -35,18 +35,13 @@ export class FuzzySearchComponent {
   displayWithFn: (MetadataRecord) => string = (record) => record?.title
 
   autoCompleteAction = (query) =>
-    this.esService
-      .buildAutocompletePayload(query)
+    this.searchApiService
```

Comment:
This is not very secured, but it wasn't the case before either.
Note that if a request takes a long time to respond, the results will erase more recent ones.

Reply:
I added `take(1)` here. Does that make sense?